tla/ccfraft.tla

```diff
@@ -326,6 +328,16 @@ NextConfigurationIndex(server) ==
     LET dom == DOMAIN configurations[server]
     IN Min(dom \ {Min(dom)})
 
+\* The configurations for a server up to (and including) a given index
+\* Useful for rolling back configurations when the log is truncated
+ConfigurationsToIndex(server, index) ==
+     RestrictPred(configurations[server], LAMBDA c : c <= index)
+
+\* Index of the last reconfiguration before the given index,
+\* assuming the given index is after the commit index
+LastConfigurationBeforeIndex(server, index) ==
+    Max({c \in DOMAIN configurations[server] : c <= index})
```

Comment:
The comment says "before" but the check is less-or-equal.

Reply:
Replaced with "up to and including"